paddle/fluid/operators/CMakeLists.txt

```diff
@@ -48,8 +48,13 @@ if (WITH_DISTRIBUTE)
     SET(OP_PREFETCH_DEPS ${OP_PREFETCH_DEPS} parameter_prefetch)
 endif()
 
+SET(OP_ONLY_MKL "")
+if (NOT WITH_MKL)
+    SET(OP_ONLY_MKL ${OP_ONLY_MKL} var_conv_2d_op)
+endif()
```

Comment:
line51-54 is not elegant.
if (NOT WITH_MKL), `var_conv_2d_op` should not be compiled and added into `GLOB_OP_LIB `

Maybe you should write the following instead.
```
if (WITH_MKL)
   op_library(var_conv_2d_op) 
endif()
```

Reply:
I tried your code, but it seems not work. Also, I tried this, still don't work:
```
if (WITH_MKL)
    op_library(var_conv_2d_op)
    file(APPEND ${pybind_file} "ONLY_AVLID_WITH_MKL(var_conv_2d);\n")
endif()
```